client/src/components/RequestForm.js

```diff
@@ -1,83 +1,139 @@
 import styled from "@emotion/styled";
 import React from "react";
-import colors from "../utils/colors";
+import PropTypes from "prop-types";
+import { Dropdown } from "./Dropdown";
+import { Input } from "./Input";
+import { FloatingLabel } from "./FloatingLabel";
+import { ButtonFull } from "./ButtonFull";
+import { Textarea } from "./Textarea";
+
+export default function RequestForm() {
+  const [artist, setArtist] = React.useState("");
+  const [date, setDate] = React.useState("");
+  const [location, setLocation] = React.useState("");
+  const [city, setCity] = React.useState("");
+  const [arrival, setArrival] = React.useState("");
+  const [price, setPrice] = React.useState("");
+  const [text, setText] = React.useState("");
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const request = {
+      artist,
+      date,
+      location,
+      city,
+      arrival,
+      price,
+      text,
+    };
+
+    alert(JSON.stringify(request));
+  }
 
-export default function ConcertCardBig() {
   return (
-    <Form type="submit">
-      <Div>
-        <Label>Band/Künstler</Label>
-        <Input type="text" placeholder=""></Input>
-      </Div>
-      <Div>
-        <Label>Datum</Label>
-        <Input type="date" placeholder=""></Input>
-      </Div>
-      <Div>
-        <Label>Location</Label>
-        <Input type="text" placeholder=""></Input>
-      </Div>
-      <Div>
-        <Label>Stadt</Label>
-        <Input type="text" placeholder=""></Input>
-      </Div>
-      <Div>
-        <Label>Anfahrt</Label>
-        <Input type="text" placeholder=""></Input>
-      </Div>
-      <Div>
-        <Label>Ticketpreis</Label>
-        <Input type="text" placeholder=""></Input>
-      </Div>
-      <Div>
-        <Label>Dein Text</Label>
-        <Input type="text" placeholder=""></Input>
-      </Div>
+    <Form onSubmit={handleSubmit}>
+      <Wrapper>
+        <Input
+          required
+          value={artist}
+          onChange={(event) => {
+            setArtist(event.target.value);
+          }}
+        />
+        <FloatingLabel>Band/Künstler</FloatingLabel>
+      </Wrapper>
```

Comment:
Since all `Input` components use a `Wrapper`, you could just include it with the component

Reply:
I did it like that but it doesn't work with my floating label :-/ But I keep that in mind and maybe adjust it later